sorc/checkout.sh

```diff
@@ -87,7 +87,7 @@ if [[ ! -d gsd_prep_chem.fd ]] ; then
     rc=$?
     ((err+=$rc))
     cd gsd_prep_chem.fd
-    git checkout gefs_v12.2.0
+    git checkout gefs_v12.3.2
```

Comment:
I think the tag Sam created is [gefs.v12.3.2](https://github.com/NOAA-GSL/GSL-prep-chem/releases/tag/gefs.v12.3.2). Please double check

Reply:
you are right.